extension/js/common/composer.ts

```diff
@@ -625,6 +624,9 @@ export class Composer {
     if (String(this.S.cached('input_to').val()).length) { // evaluate any recipient errors earlier treated as gentle
       await this.parseRenderRecipients('harshRecipientErrs');
     }
+    if (this.S.cached('icon_show_prev_msg').hasClass('progress')) {
+      throw new ComposerNotReadyError('Retrieving previous message, please wait.');
```

Comment:
Whoops I deleted the previous comment. Thanks for following the surrounding code style, even if checking state from DOM is not considered the best practice, it has worked for me.